do/sandbox.go

```diff
@@ -1,23 +1,73 @@
 package do
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
+	"net/http"
 	"os"
 	"os/exec"
+	"strings"
+
+	"github.com/digitalocean/godo"
 )
 
-// SandboxService is an interface for interacting with the sandbox plugin.
+// SandboxCredentials is the type returned by the ResolveToken and ResolveNamespace functions
+// The values in it can be used to connect sandbox support to a specific namespace using the plugin.
+type SandboxCredentials struct {
+	Auth    string
+	ApiHost string
+}
+
+// The type of the "namespace" member of POST input for API calls.
+// Only one of the fields is typically specified
+type inputNamespace struct {
+	Token     string `json:"token,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
+}
+
+// The type of the "namespace" member of the response to API calls.  Only some
+// fields are relevant to each call.
+type outputNamespace struct {
+	ApiHost   string `json:"api_host"`
+	Uuid      string `json:"uuid"`
```

Comment:
```suggestion
	APIHost   string `json:"api_host"`
	UUID      string `json:"uuid"`
```

Go is opinionated about how "initialisms" should be capitalized. 

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Unfortunately, this code base does not currently pass `golint` but it would be great to follow its advice for new code. 

```
$ golint do/...
do/sandbox.go:19:2: struct field ApiHost should be APIHost
do/sandbox.go:32:2: struct field ApiHost should be APIHost
do/sandbox.go:33:2: struct field Uuid should be UUID
do/sandbox.go:182:1: context.Context should be the first parameter of a function
do/sandbox.go:199:1: context.Context should be the first parameter of a function
```

Reply:
Fixed these, and the `do` directory now passes `golint`.

I notice that it still fails in the `commands` directory.   In the spirit of "follow its advice for new code,"  I plan to do a small follow-on PR to fix lint errors in all the files that are "new" as of `feature/sandbox`.   Not in this PR, though.